.github/workflows/apply.yaml

```
@@ -21,8 +25,24 @@ on:
+  # group so every run of this workflow -- manual or automatic, any ref --
+  # is always serialized against every other.
+  group: ${{ github.workflow }}
   cancel-in-progress: false
+  # Default queue behavior only keeps the single most-recently-queued run
+  # pending in a group -- an older pending run gets canceled and replaced.
+  # A manual one-time recovery run (state rm, or the exclude_addresses
+  # escape hatch) dispatched while a scheduled/push run is in progress
+  # could get silently dropped and replaced by the next automatic trigger
+  # before it ever executes. queue: max keeps every pending run queued
+  # (up to GitHub's cap of 100) instead of dropping older ones.
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)